client/modules/moduleRoutes.js

```diff
@@ -0,0 +1,5 @@
+//this will facilitate adding routes via the CLI
+
+export { default as user } from './user/userRoutes.js.jsx';
+export { default as post } from './post/postRoutes.js.jsx';
+export { default as product } from './product/productRoutes.js.jsx';
```

Comment:
Nice,  if we want to adminRoutes by default (good for users for example), what do you think of doing either 
```
... 
export { userRoutes as user } from './user/userRoutes.js.jsx';
export { userAdminRoutes as userAdmin } from './user/userRoutes.js.jsx'; 
... 
```
or alternatively we could do them in different files? 
```
...
export { default as user } from './user/userRoutes.js.jsx';
export { defaule as userAdmin } from './user/userAdminRoutes.js.jsx';
...
```